python/graphstorm/sagemaker/sagemaker_partition.py

```diff
@@ -24,15 +24,20 @@
 import time
 import subprocess
 from threading import Thread, Event
+from typing import List
 
-import numpy as np
 import boto3
+import botocore
+import numpy as np
 import sagemaker
+from joblib import Parallel, delayed # pylint: disable=wrong-import-order
```

Comment:
If the pylint error is caused by the line 18? If we put the ``from dataclasses import dataclass`` under the ``from typing import List``, it may solve the pylint problem.

Reply:
Nope it's from the line above importing `sagemaker`, see https://github.com/awslabs/graphstorm/issues/881